test/test.js

```diff
@@ -36,3 +40,41 @@ test('sync - cwd option', t => {
 		package_,
 	);
 });
+
+test('sync - normalize option', async t => {
+	const package_ = readPackageSync({normalize: false});
+	t.is(package_.name, 'unicorn ');
+});
+
+const pkgJson = {
+	name: 'unicorn ',
+	version: '1.0.0',
+	type: 'module',
+};
+
+test('parsePackage - json input', t => {
+	const package_ = parsePackage({...pkgJson});
```

Comment:
Since the object passed to `parsePackage` is then passed to `normalizePackageData`, this function has side-effects and will modify the original object. Should `parsePackage` spread any objects it receives?

Reply:
The input object should never be modified. Spread is not enough. So I suggest using `structuredClone` on Node.js 18 and then just spread on older Node.js versions with a note about the behavior.